src/app/shared/object-list/my-dspace-result-list-element/item-list-preview/item-list-preview.component.html

```diff
@@ -33,7 +33,7 @@ <h2 [innerHTML]="dsoTitle" [ngClass]="{'lead': true,'text-muted': !item.firstMet
           </span>
 
           <ds-truncatable-part [id]="item.id" [minLines]="1" class="item-list-abstract">
-            <span [ngClass]="{'text-muted': !item.firstMetadataValue('dc.description.abstract')}"
+            <span [ngClass]="{'text-muted': !item.hasMetadata('dc.description.abstract')}"
               [innerHTML]="(item.firstMetadataValue('dc.description.abstract')) || ('mydspace.results.no-abstract' | translate)"></span>
```

Comment:
Same here, I think this needs to be updated to `item.firstMetadataValue('dc.description.abstract', undefined, true)` in order to align with #4737.  Currently, I find the abstract is **not escaped** in the results list of the MyDSpace page.  I believe this is why that's occurring.

I can reproduce this by adding an HTML link to the abstract, like `This abstract includes a link to the <a href="https://dspace.org">DSpace website</a>.`